src/Console/ConsoleOptionParser.php

```diff
@@ -298,7 +298,7 @@ public function getCommand()
      *
      * @deprecated 3.4.0 Use setCommand()/getCommand() instead.
      * @param string|null $text The text to set, or null if you want to read
-     * @return string|$this If reading, the value of the command. If setting $this will be returned.
+     * @return ConsoleOptionParser|string If reading, the value of the command. If setting $this will be returned.
```

Comment:
In these cases $this is correct though afaik - the IDE knows that this is chaining and knows the type implicitly.